### PR TITLE
Feature/vdx 68/save vc

### PR DESCRIPTION
**Pull request by mehmetbicak**

### CODE REVIEW

**Review by sksadjad — CHANGES_REQUESTED** [review]
add a few notes, but couldn't add another one on a new empty file that you've created "memory". do we need it?

File: packages/ms-request-api/package.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
 {
   "name": "@sphereon/ms-request-api",
   "version": "0.6.0",
```
[the remark this answers is not in the document]
Comment (mehmetbicak) [reply packages/ms-request-api/package.json:4]: We have to only change the main version (ssi-sdk/package.json) The others are automatically changed.

File: packages/ms-request-api/__tests__/shared/msRequestApiAgentLogic.ts [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 import { fetchIssuanceRequestMs } from '../../src/IssuerUtil'
 import { IMsRequestApi, IIssueRequestResponse, IClientIssueRequest, IClientIssuanceConfig } from '../../src/types/IMsRequestApi'
-import { TAgent } from '@veramo/core'
 import { v4 as uuidv4 } from 'uuid'
+import { createAgent, TAgent , IDataStore, IDataStoreORM, VerifiableCredential, FindArgs, TCredentialColumns} from '@veramo/core'
+import { Entities, DataStore, DataStoreORM } from '@veramo/data-store'
+import { createConnection } from 'typeorm'
 
-type ConfiguredAgent = TAgent<IMsRequestApi>
+type ConfiguredAgent = TAgent<IMsRequestApi & IDataStore & IDataStoreORM >;
```
[the remark this answers is not in the document]
Comment (mehmetbicak) [reply packages/ms-request-api/__tests__/shared/msRequestApiAgentLogic.ts]: Reformatted

**Comment by mehmetbicak** [comment]
Deleted the unnecessary empty memory file. Also added a code to close the db connection after test is done.

**Review by sksadjad — APPROVED** [review]
looks good to me

File: packages/ms-request-api/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (nklomp) [reply packages/ms-request-api/package.json:4]: Next time don;'t even change the main version. As mentioned, this is handled for us. No need to change any version anywhere

**Pull request merged**